BUILD

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 The Marl Authors.
```

Comment:
Please can we rename this file to `BUILD.bazel`?
This collides with the default `build` output directory on case insensitive filesystems.